orienteer-core/src/main/java/org/orienteer/core/widget/command/FullScreenCommand.java

```diff
@@ -33,4 +32,9 @@ public void onClick(AjaxRequestTarget target) {
 		target.appendJavaScript("$('body').toggleClass('noscroll'); $('#"+widget.getMarkupId()+"').toggleClass('fullscreen');");
 	}
 
+	@Override
+	protected void onDetach() {
+		this.expanded = false;
```

Comment:
Not sure that this is right. You might work with expanded window in a set of requests-responses.